Wordle.Api/Wordle.Api/Identity/Policies.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace Wordle.Api.Identity;
+public static class Policies
+{
+    public const string RandomAdmin = "RandomAdmin";
+
+    public static void RandomAdminPolicy(AuthorizationPolicyBuilder policy)
+    {
+        policy.RequireRole(Roles.Admin);
+        policy.RequireAssertion(context =>
+        {
+            var random = context.User.Claims.FirstOrDefault(c => c.Type == Claims.Random);
+            if (Double.TryParse(random?.Value, out double result))
+            {
+                return result > 0.5;
+            }
+            return false;
+        });
+    }
```

Comment:
```suggestion
    public static void EditWordPolicy(AuthorizationPolicyBuilder policy)
    {
        policy.RequireClaim(Claims.MotU);
        policy.RequireAssertion(context =>
        {
            var ageString = context.User.Claims.FirstOrDefault(f => f.Type == Claims.Age);
            if (ageString != null)
            {
                int age;
                if (int.TryParse(ageString.Value, out age))
                {
                    return age >= 21;
                }
            }
            return false;
        });
    }
```
Replaced with in-class example.
This policy was just an example policy. We should create a claim for birthday / masteroftheuniverse like we did in class.

Reply:
Fixed